setup.py

```diff
@@ -3,21 +3,27 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-# TODO Adapt your project configuration to your own project.
-# The name of the package is the one to be used in runtime.
-# The 'install_requires' is where you specify the package dependencies of your package. They will be automaticly installed, before your package.  # noqa: E501
+requirements = [
```

Comment:
The requirements are duplicated, as is, this variable is not needed

Reply:
@erezz-mov-ai or @engdorm can you remove duplication please